tests/integration/test_charm.py

```diff
@@ -28,3 +37,66 @@ async def test_build_and_deploy(ops_test: OpsTest):
     await ops_test.model.wait_for_idle(apps=["kafka", "zookeeper"])
     assert ops_test.model.applications["kafka"].status == "active"
     assert ops_test.model.applications["zookeeper"].status == "active"
+
+
+@pytest.mark.abort_on_fail
+async def test_logs_write_to_storage(ops_test: OpsTest):
+    app_charm = await ops_test.build_charm("tests/integration/app-charm")
+    await asyncio.gather(
+        ops_test.model.deploy(app_charm, application_name=DUMMY_NAME, num_units=1, series="jammy"),
+    )
```

Comment:
Not at all, I think it's there because I copied it from the previous :).

The `gather` just bundles together various `awaitable`s to run concurrently, the same would be achieved with `await` for a single corroutine/Task/Future.